specs/2024-001.json

```diff
@@ -21,15 +21,17 @@
             ],
             "optional": [
                 "AIUsage.md",
-                "Feedback.md"
+                "Feedback.md",
+                "*.png"
```

Comment:
```suggestion
```

See my other comment - I'd prefer to have false negatives rather than potentially miss flagging genuinely un-used files.

Also, due to some oddities with how the `optional` and `data-type-patterns` keys work, if we were to include `*.png` it needs to go in the `data-type-patterns` block (otherwise if there are 2 extra `.png` files, the checker will only be OK with the first one!)